source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,127 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
 B. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Proin non mi a felis luctus vulputate nec pharetra odio. Ut pretium
-scelerisque nulla, sed euismod nisi pellentesque sit amet. Proin mollis
-mauris eu libero ullamcorper rutrum. Pellentesque vitae enim sed magna
-accumsan maximus et quis mi. Ut vel laoreet turpis. Vivamus vitae odio
-sed arcu finibus pretium vel maximus nisi. Nullam gravida consequat
-porta.
+An important concept in database interaction is `data validation 
```

Comment:
- Let's de-emphasize the data validation aspect of JSON schema. We should note that the use case and process of defining a validation schema is different than FLE so that our users are not confused, but otherwise we should focus on encryption (I realize this is a shift from some of my earlier comments)
- Let's emphasize why we use JSON schema to configure encryption (i.e. answer the question: "why did MongoDB engineer the feature this way?") and give some quick insight into how it works under the hood without giving the full technical breakdown.

Reply:
@ccho-mongodb @nlarew 

Thanks for your suggestions with this. I've created a brief paragraph that I think satisfies the requirements of: 
* explaining briefly what data validation and JSON schema briefly and linking to a doc with further explanation
* not using reductive phrases (like 'important' or 'simple')
* using active voice

Paragraph:

B. Define a JSON Schema

"While designing your database you may want to enforce specific datatypes on specific fields. MongoDB works with JSON Schema to reduce the complexity in data validation during insert and update operations. Within the context of Field Level Encryption, JSON Schema is used to specify several configuration details, allowing the drivers to automatically encrypt and decrypt fields."

What do you think?

Reply:
This version is much better, but could still be more direct. Examples:
1) "works with JSON Schema to reduce..." - could be revised to explain how it uses JSON Schema rather than just "working with" it.
2) "within the context of..." - we could instead use this opportunity to mention that it is required rather than that it has some meaning within the context of CSFLE.

Reply:
Nitpick: "specific datatypes on specific fields" seems repetitive and maybe should only be used once.

Reply:
Good ideas. I've updated it, I think you'll like the updated paragraph. It resolves your two points and the nitpick.

Reply:
@ccho-mongodb

Reply:
This thread got marked outdated, so will comment on the new paragraph.